config.json

```diff
@@ -1307,6 +1307,14 @@
         ],
         "difficulty": 3
       },
+      {
+        "slug": "bottle-song",
+        "name": "Bottle Song",
+        "uuid": "6c240d6c-d897-4c4e-a556-b595f45c2d3b",
+        "prerequisites": [],
+        "practices": [],
```

Comment:
We could [copy the list](https://github.com/exercism/elixir/pull/1356/files#diff-587cb980af76fdc7e52369fd0b9d926dff266976b6f8ac631e358fecc49ff8cfL1238-L1250) from `beer-song`, its predecessor.